app/Http/Controllers/SubscriptionPaymentController.php

```
@@ -0,0 +1,647 @@
+                'user_id' => $activeSubscription->user_id,
+                'tenant_id' => $activeSubscription->tenant_id,
+                'status' => $activeSubscription->status,
+                'expires_at' => $activeSubscription->expires_at,
+            ] : null,
+        ];
+    }
+
+    /**
+     * Manual process payment for debugging
+     */
+    public function manualProcessPayment(Request $request)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)